src/meta/rule.schema.json

```diff
@@ -1,18 +1,23 @@
 {
-  "$id": "http://stoplight.io/schemas/rule.schema.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/stoplightio/spectral/blob/develop/src/meta/rule.schema.json",
```

Comment:
Seeing as $id does not need to resolve, could we a) remove this, or b) set it to something that we control so if people do $ref using it its stable? `http://stoplight.io/schemas/rule.schema.json` with a redirect would be ideal if we need to make it resolve for some reason.

Reply:
yeah, b) would be ideal. I'll try to see if that's possible.

Reply:
Also maybe namespace under spectral. `http://stoplight.io/schemas/spectral/rule.json` so other tools can define schemas.